build/fileindex.js

```diff
@@ -4,35 +4,33 @@ module.exports = {
             format: 'json_flat',
             pretty: true
         },
-        files: [{ 
-            dest: global.dist + '/javascript.json', 
+        files: [{
+            dest: global.dist + '/javascript.json',
             src: [
-            'src/javascript/lib/jquery.js', 
-            'src/javascript/lib/highstock/highstock.js', 
+            'src/javascript/lib/jquery.js',
+            'src/javascript/lib/highstock/highstock.js',
             'src/javascript/lib/highstock/highstock-exporting.js',
-            'src/javascript/lib/moment/moment.js', 
+            'src/javascript/lib/moment/moment.js',
             'src/javascript/lib/**/*.js',
```

Comment:
Does this include all `.js` files in the `lib/`? If so we can remove the above 4 lines.